src/components/toggle.jsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+import HandleView from './view.jsx';
+
+class ToggleContainerViews extends React.Component {
+  constructor(props) {
+    super();
+
+    const view = props.defaultView;
+
+    this.state = {
+      view,
+    };
+  }
+
+  handleClick(view) {
+    this.setState({
+      view,
```

Comment:
Okay, I'll make an alternative with that approach. Then we can decide which one to use 👯